OpenRA.Mods.Common/Widgets/LabelWidget.cs

```diff
@@ -28,15 +30,42 @@ public class LabelWidget : Widget
 		public bool Contrast = ChromeMetrics.Get<bool>("TextContrast");
 		public Color ContrastColor = ChromeMetrics.Get<Color>("TextContrastColor");
 		public bool WordWrap = false;
+
+		[Desc("Space between lines as a percentage (default) of line height or fixed pixel amount (rounded down).")]
+		public int LineSpacing = 40;
```

Comment:
Convention for line spacing (see e.g. CSS, Latex, MS Word, etc) is to specify the *full* line height, so please change this to `public float LineSpacing = 1.4f;`. This remains compatible with `LineSpacingType.Fixed`.